x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/type/StringUtils.java

```diff
@@ -26,7 +26,7 @@
 
 //FIXME: this class comes from sql-proto
 // find a way to share it across or potentially just copy it over
```

Comment:
I guess this could be removed now too (in case of any further changes go in, not worth another CI round).